# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1549302620 0
#      Mon Feb 04 17:50:20 2019 +0000
# Branch stable
# Node ID 8c50b03c2016a79ffcfcbfca0e24a8dabf382771
# Parent  af47f245a2ada829fa87f339bbe8e7c64a189bda
Added tag rc-5-0-91 for changeset af47f245a2ad

diff --git a/.hgtags b/.hgtags
--- a/.hgtags
+++ b/.hgtags
@@ -110,8 +110,9 @@ 708a4fcb73820484a3af94dd4d18453c4d9b29c9
 1327ea4f5a931a6cc2a6d4790af58cf16de4702b release-4-2-1
 94f8e2b5a88b79974084c0ae541cacbcc62d64f6 release-4-2-2
 a5ad9c053e41ed39a8c69ff99936cbcf18256abf rc-4-3-90
 ff59be58f0282e418fb3447d5442fa1d618cfe10 rc-4-3-91
 4c14c002cabeb60cf693d5a032b6285f92804643 rc-4-3-92
 cf84db75ab57d4fc5d18682c6ecf9d2e1f66ea94 release-4-4-0
 1f46d371968c5c2e9d5fac537046db8005b71e41 release-4-4-1
 82865ccb62c2aa2f1273683b8343a80e2e6a0e83 rc-5-0-90
+af47f245a2ada829fa87f339bbe8e7c64a189bda rc-5-0-91
